src/System.IO.FileSystem/src/System/IO/UnixFileSystem.cs

```diff
@@ -402,11 +402,11 @@ private sealed class FileSystemEnumerable<T> : IEnumerable<T>
                 // Basic validation of the input path
                 if (userPath == null)
                 {
-                    throw new ArgumentNullException("path");
+                    throw new ArgumentNullException(nameof(userPath));
                 }
                 if (string.IsNullOrWhiteSpace(userPath))
                 {
-                    throw new ArgumentException(SR.Argument_EmptyPath, "path");
+                    throw new ArgumentException(SR.Argument_EmptyPath, nameof(userPath));
```

Comment:
Please revert the changes to this file.  There's a reason these weren't changed previously: the public entry point uses "path", not "userPath", as the variable name.